app/main.py

```diff
@@ -1,34 +1,77 @@
 class Deck:
-    def __init__(self, row, column, is_alive=True):
-        pass
+    def init(self, row: int, column: int, is_alive: bool = True) -> None:
+        self.row = row
+        self.column = column
+        self.is_alive = is_alive
+
+    def str(self) -> None:
+        return "□" if self.is_alive else "x"  # "□" for alive, "x" for drowned
 
 
 class Ship:
-    def __init__(self, start, end, is_drowned=False):
-        # Create decks and save them to a list `self.decks`
-        pass
+    def init(self, start: int, end: int, is_drowned: bool = False) -> None:
```

Comment:
The method `init` should be `__init__` to properly define the constructor for the `Ship` class.